### PR TITLE
Splitting up groups by "/" delimiter

### PR DESCRIPTION
**Pull request by danielkoh94**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java [hunk 1]
```diff
@@ -324,10 +324,19 @@ private List<CorpGroupSnapshot> extractGroups(CommonProfile profile) {
             groupNames = Collections.emptyList();
           }
 
-          for (String groupName : groupNames) {
+          Collection<String> splitGroupNames = new ArrayList<String>();
+
+          // Split groups that are delimited by /
+          for (String groupName: groupNames) {
+            Collection<String> splitGroups = Arrays.stream(groupName.split("/"))
+                                                   .filter(name -> name.length() > 0)
```
Comment (neojunjie) [inline datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java:332]: small point. Maybe you can check for empty space after splitting the array. name.trim().length ..

File: datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java [hunk 2]
```diff
@@ -324,10 +324,19 @@ private List<CorpGroupSnapshot> extractGroups(CommonProfile profile) {
             groupNames = Collections.emptyList();
           }
 
-          for (String groupName : groupNames) {
+          Collection<String> splitGroupNames = new ArrayList<String>();
+
+          // Split groups that are delimited by /
+          for (String groupName: groupNames) {
```
Comment (neojunjie) [inline datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java:330]: Should you check for empty groupname? Skip the whole logic if it is true

**Review by neojunjie — COMMENTED** [review]
Overall not much issue. Pretty clean logic.. Lets merge it once the comments are checked for

File: datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java (hunk 1 above)
Comment (danielkoh94) [reply datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java]: Added trim() to the filter to remove groups with empty spaces as names

**Review by neojunjie — APPROVED** [review]

**Pull request merged**